.github/codejson/cookiecutter.json

```diff
@@ -1,67 +1,54 @@
-{
-    "project_name": "{{ cookiecutter.project_name }}", 
-    "project_repo_name": "{{ cookiecutter.project_repo_name }}",
-    "project_org": "{{ cookiecutter.project_org }}", 
+{ 
+    "name": "{{ cookiecutter.project_name }}",
     "description": "A short description of the project.",
-    "long_description": "A longer description of the project.",
     "status": ["ideation", "development", "alpha", "beta", "release candidate", "production", "archival"],
     "license": ["CC0-1.0", "Apache-2.0", "MIT", "MPL-2.0", "GPL-2.0-only", "GPL-3.0-only", "GPL-3.0-or-later", "LGPL-2.1-only", "LGPL-3.0-only", "BSD-2-Clause", "BSD-3-Clause", "EPL-2.0", "Other"],
-    "usage_type" : ["openSource", "governmentWideReuse", "exemptByLaw", "exemptByNationalSecurity", "exemptByAgencySystem", "exemptByAgencyMission", "exemptByCIO", "exemptByPolicyDate"],
-    "repository_host": ["github.com/CMSgov", "github.com/CMS-Enterprise", "github.com/DSACMS", "github.cms.gov", "CCSQ GitHub"],
+    "license_URL": "",
+    "organization": "",
+    "version": "",
+    "usage_type" : ["openSource", "governmentWideReuse", "exemptByNationalSecurity", "exemptByNationalIntelligence", "exemptByFOIA", "exemptByEAR", "exemptByITAR", "exemptByTSA", "exemptByClassifiedInformation", "exemptByPrivacyRisk", "exemptByIPRestriction", "exemptByAgencySystem", "exemptByAgencyMission", "exemptByCIO", "exemptByPolicyDate"],
+    "repositoryURL": "",
+    "homepageURL": "",
+    "downloadURL": "",
+    "disclaimerURL": "",
+    "disclaimerText": "",
     "repository_visibility": ["public", "private"],
     "vcs": ["git", "hg", "svn", "rcs", "bzr"],
-    "forks": 0,
-    "platforms": "web, windows, mac, linux, ios, android, other",
-    "categories": "healthcare",
-    "software_type":["standalone/mobile", "standalone/iot", "standalone/desktop", "standalone/web", "standalone/backend", "standalone/other", "addon", "library", "configurationFiles"],
+    "laborHours": 0,
     "languages": "",
     "maintenance": ["internal", "contract", "community", "none"],
     "contract_number": 0,
-    "tags": "dsacms-tier1",
-    "contact_email": "opensource@cms.hhs.gov",
-    "contact_name": "CMS Open Source Program Office",
+    "tags": "",
+    "contact_email": "",
+    "contact_name": "A point of contact is needed for the project. What is the name of the point of contact?",
```

Comment:
Make this an empty string since this is a default value, questions belong in the `__prompts__` object